### PR TITLE
Small fixes2

### LINKED ISSUE #267: 'Non existing fund' message for /fund/0x... URL with addresses that don't correspond to an actual fund

### PR DESCRIPTION
**Pull request by jennazenk**
closes #267 and #268

### CODE REVIEW

File: imports/ui/components/portfolio/manageParticipation.js [hunk 1]
```diff
@@ -38,6 +42,18 @@ Template.manageParticipation.helpers({
       return web3.fromWei(doc.sharePrice, 'ether');
     }
   },
+  getSharePrice() {
+    const template = Template.instance();
```
Comment (schmidsi) [inline imports/ui/components/portfolio/manageParticipation.js]: Please name the instance instance:
```const instance = Template.instance();```

File: imports/ui/components/portfolio/manageParticipation.js [hunk 2]
```diff
@@ -38,6 +42,18 @@ Template.manageParticipation.helpers({
...
+    const vaultAddress = FlowRouter.getParam('address');
+    const vaultId = Vaults.findOne({ address: vaultAddress }).id;
+    Vault.setProvider(web3.currentProvider);
+    const vaultContract = Vault.at(vaultAddress);
+    vaultContract.performCalculations().then((result) => {
```
Comment (schmidsi) [inline imports/ui/components/portfolio/manageParticipation.js]: It's really dangerous to call expensive functions like this in a reactive template helper. They can get called several times in a second.

Better: Put the state in redux and just get the values from there

File: imports/ui/components/portfolio/manageParticipation.js [hunk 3]
```diff
@@ -38,6 +42,18 @@ Template.manageParticipation.helpers({
...
+      template.sharePrice.set(convertFromTokenPrecision(result[5].toNumber(), 18));
+      Meteor.call('vaults.syncVaultById', vaultId)
```
Comment (schmidsi) [inline imports/ui/components/portfolio/manageParticipation.js]: Same here: This brings the server down.

File: imports/ui/components/portfolio/manageParticipation.js (hunk 1 above)
Comment (jennazenk) [inline imports/ui/components/portfolio/manageParticipation.js]: We decide to go with `const template = Template.instance();` from now on

**Pull request merged**